AppInstallerFileBuilder/AppInstallerFileBuilder/Views/GenerateXMLView.xaml.cs

```diff
@@ -444,7 +445,13 @@ private async void Generate_File_Button_Click(object sender, RoutedEventArgs e)
                         //DataContractSerializer onLaunchDCS = new DataContractSerializer(typeof(OnLaunch));
                         //onLaunchDCS.WriteStartObject(xdw, onLaunch);
                         xdw.WriteStartElement("OnLaunch");
-                        xdw.WriteAttributeString("HoursBetweenUpdateChecks", onLaunch.HoursBetweenUpdateChecks.ToString());
+
+                        //HoursBetweenUpdate checks is only available AFTER 1709
+                        if(!App.AppInstallerFileSchemaNamespace.Equals("http://schemas.microsoft.com/appx/appinstaller/2017"))
```

Comment:
>AppInstallerFileSchemaNamespace [](start = 32, length = 31)

why is checking the schema necessary for HoursBetweenUpdateChecks but not the bools etc. below?

Reply:
HoursBetweenUpdateChecks was being handled as a default. The other bools below contain the schema checks.